driver-pravega/src/main/java/io/openmessaging/benchmark/driver/pravega/PravegaBenchmarkTransactionProducer.java

```diff
@@ -64,7 +67,10 @@ public PravegaBenchmarkTransactionProducer(String streamName, EventStreamClientF
     public CompletableFuture<Void> sendAsync(Optional<String> key, byte[] payload) {
         try {
             if (transaction == null) {
+                this.stateChangedCommitting = 0;
                 transaction = transactionWriter.beginTxn();
+                // Start measurement - wait until transaction is OPEN
+                this.stateChangedCommitting = this.getTimeStatusReached(transaction, Transaction.Status.OPEN);
```

Comment:
The line is not needed - `transaction` is in OPEN stated after beginTxn() is called. (it's sync method)